basyx.submodelrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/submodelrepository/http/TestSubmodelRepositorySubmodelElementsHTTP.java

```diff
@@ -222,6 +225,67 @@ public void setSubmodelElementValueOfNonExistingSubmodel() throws IOException {
 		assertEquals(HttpStatus.NOT_FOUND.value(), response.getCode());
 	}
 
+	@Test
+	public void createExistingSubmodelElement() throws IOException {
+		String element = getSubmodelElementJSON("SubmodelElement.json");
+		CloseableHttpResponse createdResponse = BaSyxHttpTestUtils.executePostOnServer(createSubmodelElementsURL(DummySubmodelFactory.SUBMODEL_TECHNICAL_DATA_ID), element);
+		
+		assertEquals(HttpStatus.CONFLICT.value(), createdResponse.getCode());
+	}
+	
+	@Test
+	public void createDeleteSubmodelElement() throws FileNotFoundException, IOException, ParseException {
+		String element = getSubmodelElementJSON("SubmodelElementTobeDeleted.json");
+		createSubmodelElementHelper("ElementToDelete", element);
```

Comment:
idShort in eigene Konstante auslagern und dann überall wiederverwenden

Reply:
checked